# Question 27
-----

In [1]:
import math
def P(AGE, SMK, SEX, CHOL, OCC):
    PX = 1 / (1 + math.exp(-1 * (-4.32 + (0.0274*AGE) + (0.5859*SMK) + (1.1523*SEX) + (0.0087*CHOL) + (-0.5309*OCC))))
    return PX

smoker = P(40, 1, 1, 200, 1)
non_smoker = P(40, 0, 1, 200, 1)
risk_ratio = smoker / non_smoker
print(f"risk ratio: {risk_ratio}")

risk ratio: 1.4530599917805596


Hence, smokers have 1.45 times increased risk of hypertension.